ChessClock/Clocks/ClocksPresenter.swift

```diff
@@ -18,6 +18,8 @@ enum ClocksEvents {
     case resume
 }
 
+let kStartingTime = 500
```

Comment:
what does the "k" stand for? 🤔

Reply:
hmmm good question! I have just been using it to denote constant unconsciously. I think it's almost like a legacy habit that stayed in iOS since the objective-c time. https://stackoverflow.com/a/34995419

Admittedly it's a little dated.

Reply:
Ok, so maybe we can change the name so that we don't use the "k". Anyway I think this parameter should be injected into the presenter, so in the future the user can select the starting time from a settings screen or similar. (that will come in other PRs tho! :)